modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -107,9 +109,13 @@ ifdef::virt-importing-vmware-vm[]
 If you try to import a virtual machine with a disk that is larger than the available storage space, the operation cannot complete. You will not be able to import another virtual machine or to clean up the storage because there are insufficient resources to support object deletion. To resolve this situation, you must add more object storage devices to the storage backend. link:https://bugzilla.redhat.com/show_bug.cgi?id=1721504[(*BZ#1721504*)]
 endif::[]
 ifdef::virt-importing-rhv-vm[]
-* In {VirtProductName} 2.4, the import process fails if the target VM name exceeds 23 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
+ifeval::["{VirtVersion}" == "2.4"]
+* The import process fails if the target VM name is longer than 23 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
+
+* If you import a Windows VM, you must create a Windows template and a custom ConfigMap. link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[(*BZ#1856654*)]
+endif::[]
 
-* In {VirtProductName} 2.4, you must create a Windows {VirtProductName} template and a custom ConfigMap in order to import a RHV VM with a Windows operating system.
+* If you import a Linux VM that does not have a corresponding {product-title} template, you must create a custom ConfigMap.
```

Comment:
Please remove the "Linux". and again, if a template is missing in CNV need to add a custom template, and also a configMap that will map VM OS to it.

Reply:
We do not provide instructions for creating a Linux custom template. Do you plan to add a procedure to create one?

Reply:
We do not suppose to provide in VM import "how to create a template" instructions. I mentioned earlier in this review the link to where it is in the CNV/Virt doc section.

There is no need to separate Linux & Windows.

If users want, they can add another Windows template, and not the exact one we provided as an example in the w/a.